_posts/2018-05-24-open-source-hackillinois-2018.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Open Source at HackIllinois 2018"
+date: 2018-05-24 00:00:00
+categories: recap
+tags: HackIllinois2018, featured
+image: /assets/article_images/open-source-hackillinois-2018/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and [25 sponsors](https://2018.hackillinois.org) at HackIllinois, and everyone came eager to contribute to the Open Source community! The goal of HackIllinois has expanded in recent years to connect with the developer community by encouraging students and hackers from all over the country to contribute to the tools they use every day. Hackers get flexibility as to what they want to work on—they can opt to build their own projects, or they can have the unique experience of learning how to contribute to existing Open Source software. To facilitate these contributions, the HackIllinois Team invites mentors from awesome Open Source projects to work with Hackers at the event and help them get involved.
+
+# Mentors
+
+Open Source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience: these experienced Open Source contributors and engineers help students get the ball rolling on contributing to Open Source. 
+
+To initially introduce students to the mentors, we held panels regarding Web/Data and Systems/Miscellaneous so students could come hear about the fields they were interested in. Afterward, mentors briefly described their projects and students had a chance to discuss with mentors and join their teams.
+
+![Mentors with experience in Web/Data discuss at the panel in Siebel](/assets/article_images/open-source-hackillinois-2018/panel_speakers.jpg "Web/Data panel")
+
+Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand, and were prepared to guide students through the initially intimidating process of understanding a codebase and improving it. 
```

Comment:
you do not actually need the comma after "beforehand" since there is only one subject predicate pair in the whole sentence. I see that it could possibly be useful in breaking up the sentence however, so whatever you think would be clearest to the reader is fine here, just wanted to point it out

Reply:
Wow big grammar boy LOL but kk